### ISSUE TITLE
[CLOSED] IMAP connection broken (server response)

### ISSUE THREAD

**Issue by ranjbarhadi** [ladybirdweb/faveo-helpdesk#656]
- Faveo Version : Community 1.9.6
- PHP version : 7.0
- Database Driver & Version : mysql 5.6
- Server specification :  Ubuntu 16.04 LTS (GNU/Linux 4.4.0-21-generic x86_64)

#### Description:
when I try to add an email the error `[CLOSED] IMAP connection broken (server response)` shows at the top of the page. the mail server is ok and working with another instance of Faveo.
the correct permissions are set. it appears that IMAP is not working properly even though it's installed. firewall is inactive.i've check SMTP  protocol works. but the problem is with IMAP. what should I do? I'm frustrated 

#### Downloaded from
- [x] master-branch

- [ ] development-branch

- [ ] release-tag

**Comment by ranjbarhadi** [comment]
seriously? no one?

**Comment by pemitic** [comment]
When I tried to add an email the error I got: Can not authenticate to IMAP server: [CLOSED] IMAP connection broken (authenticate). I think the problem is because there is no field "username" in form, when you adding new email, but if you try to edit an existing email field shows up and it is mandatory.

I did some workaround:
- in phpmyadmin I changed field "user_name" to allow NULL values in emails table
- I add new email in Faveo HelpDesk but not enabled "Incoming email information", it should add email
- edit email and fill "User Name" with email, enable "Incoming email information", enter needed information for IMAP server and smtp server and save it

I hope so, it will help you.
